openprescribing/frontend/management/commands/measure_definitions/WIP_enviromental_inhalers

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Enviromental Impact of Inhalers",
```

Comment:
Enviro_n_mental

Reply:
thanks and updated. I wonder how many years I have been spelling that wrong for.....